view/frontend/web/js/view/payment/method-renderer/adyen-pm-method.js

```diff
@@ -320,6 +318,12 @@ define(
                 );
             },
 
+            handleOnFailure: function(response, component) {
+                const self = this;
+                self.isPlaceOrderAllowed(true);
```

Comment:
Since execution happening in the same context, you don't need to define `self` and use it instead of `this`. Feel free to use `this` directly here.